04/01/2025 05:01:08 PM: INFO: Opening URL: https://health-demo.digit.org/microplan-ui/employee/user/login
04/01/2025 05:01:12 PM: INFO: Performing login actions
04/01/2025 05:01:14 PM: INFO: Login successful, title verified
04/01/2025 05:01:15 PM: WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5576990890>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/8c066e6a36475c2fc0bcc46c7385b2ad
04/01/2025 05:01:15 PM: WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5576990920>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/8c066e6a36475c2fc0bcc46c7385b2ad
04/01/2025 05:01:15 PM: WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5576990440>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/8c066e6a36475c2fc0bcc46c7385b2ad
04/01/2025 05:01:16 PM: INFO: Navigating to the Microplan creation
04/01/2025 05:02:15 PM: WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5576990bf0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/aecfe91a73057fdc388f5adbacf04edf
04/01/2025 05:02:15 PM: WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5576992240>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/aecfe91a73057fdc388f5adbacf04edf
04/01/2025 05:02:15 PM: WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5576992a20>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/aecfe91a73057fdc388f5adbacf04edf
04/01/2025 05:02:16 PM: INFO: Uploading population and facility details
04/01/2025 05:02:23 PM: WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5576993020>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0b31ba7b44a5316f01af1ac610f610de
04/01/2025 05:02:23 PM: WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55769875c0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0b31ba7b44a5316f01af1ac610f610de
04/01/2025 05:02:23 PM: WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5576987b30>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0b31ba7b44a5316f01af1ac610f610de
04/01/2025 05:02:23 PM: WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5576985b80>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0b31ba7b44a5316f01af1ac610f610de
04/01/2025 05:02:23 PM: WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5576985e80>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0b31ba7b44a5316f01af1ac610f610de
04/01/2025 05:02:23 PM: WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5576986000>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0b31ba7b44a5316f01af1ac610f610de
04/01/2025 05:02:31 PM: INFO: Opening URL: https://health-demo.digit.org/microplan-ui/employee/user/login
04/01/2025 05:02:35 PM: INFO: Performing login actions
04/01/2025 05:02:37 PM: INFO: Login successful, title verified
04/01/2025 05:02:37 PM: WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x74d8f37ed250>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/98581079e06caab487e11b12ef7c7f83
04/01/2025 05:02:37 PM: WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x74d8f37ed310>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/98581079e06caab487e11b12ef7c7f83
04/01/2025 05:02:37 PM: WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x74d8f37ed4c0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/98581079e06caab487e11b12ef7c7f83
04/01/2025 05:02:39 PM: INFO: Navigating to the Microplan creation
04/01/2025 05:04:17 PM: WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x74d8f37ee750>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/b1ec9d5732b9ff8366dcbb671d771abf
04/01/2025 05:04:17 PM: WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x74d8f385be00>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/b1ec9d5732b9ff8366dcbb671d771abf
04/01/2025 05:04:17 PM: WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x74d8f3859f10>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/b1ec9d5732b9ff8366dcbb671d771abf
04/01/2025 05:04:17 PM: WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x74d8f385bef0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/b1ec9d5732b9ff8366dcbb671d771abf
04/01/2025 05:04:17 PM: WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x74d8f385bd70>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/b1ec9d5732b9ff8366dcbb671d771abf
04/01/2025 05:04:17 PM: WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x74d8f3858ad0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/b1ec9d5732b9ff8366dcbb671d771abf
04/01/2025 05:07:01 PM: INFO: Opening URL: https://health-demo.digit.org/microplan-ui/employee/user/login
04/01/2025 05:07:05 PM: INFO: Performing login actions
04/01/2025 05:07:07 PM: INFO: Login successful, title verified
04/01/2025 05:07:08 PM: WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x748d62190c80>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/7c5517ecd4a47d763738c34ddeac9892
04/01/2025 05:07:08 PM: WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x748d62190d70>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/7c5517ecd4a47d763738c34ddeac9892
04/01/2025 05:07:08 PM: WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x748d62190f20>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/7c5517ecd4a47d763738c34ddeac9892
04/01/2025 05:07:09 PM: INFO: Navigating to the Microplan creation
04/01/2025 05:09:08 PM: WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x748d62190e90>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/bac527b3c29b0f5d0249e2d55ba54223
04/01/2025 05:09:08 PM: WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x748d621910d0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/bac527b3c29b0f5d0249e2d55ba54223
04/01/2025 05:09:08 PM: WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x748d62192990>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/bac527b3c29b0f5d0249e2d55ba54223
04/01/2025 05:09:09 PM: INFO: Uploading population and facility details
04/01/2025 05:09:22 PM: WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x748d62193620>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/ca205135219050070a948e4900c3a7a9
04/01/2025 05:09:22 PM: WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x748d621eba40>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/ca205135219050070a948e4900c3a7a9
04/01/2025 05:09:22 PM: WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x748d621e86b0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/ca205135219050070a948e4900c3a7a9
04/01/2025 05:09:22 PM: WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x748d621eb080>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/ca205135219050070a948e4900c3a7a9
04/01/2025 05:09:22 PM: WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x748d621ebbc0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/ca205135219050070a948e4900c3a7a9
04/01/2025 05:09:22 PM: WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x748d621e90d0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/ca205135219050070a948e4900c3a7a9
04/01/2025 05:10:48 PM: INFO: Opening URL: https://health-demo.digit.org/microplan-ui/employee/user/login
04/01/2025 05:10:52 PM: INFO: Performing login actions
04/01/2025 05:10:54 PM: INFO: Login successful, title verified
04/01/2025 05:10:54 PM: WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7906451f8d70>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/9effb4580a5f417583499f6900b9f9ba
04/01/2025 05:10:54 PM: WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7906451f8e30>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/9effb4580a5f417583499f6900b9f9ba
04/01/2025 05:10:54 PM: WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7906451f8fe0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/9effb4580a5f417583499f6900b9f9ba
04/01/2025 05:10:55 PM: INFO: Navigating to the Microplan creation
04/01/2025 05:11:47 PM: WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7906451fb290>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/6989cc1974806fcfdc4e588424acba7f
04/01/2025 05:11:47 PM: WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7906451fb3e0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/6989cc1974806fcfdc4e588424acba7f
04/01/2025 05:11:47 PM: WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7906451f9c70>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/6989cc1974806fcfdc4e588424acba7f
04/01/2025 05:11:48 PM: INFO: Uploading population and facility details
04/01/2025 05:11:53 PM: WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7906451f86b0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/ccdee1fdc44c6ca6d17e379ecd8e2127
04/01/2025 05:11:53 PM: WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x790645235700>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/ccdee1fdc44c6ca6d17e379ecd8e2127
04/01/2025 05:11:53 PM: WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7906452367b0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/ccdee1fdc44c6ca6d17e379ecd8e2127
04/01/2025 05:11:53 PM: WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x790645235790>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/ccdee1fdc44c6ca6d17e379ecd8e2127
04/01/2025 05:11:53 PM: WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x790645235a90>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/ccdee1fdc44c6ca6d17e379ecd8e2127
04/01/2025 05:11:53 PM: WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x790645235c10>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/ccdee1fdc44c6ca6d17e379ecd8e2127
